scripts/dqlite/cmd/main.go

```diff
@@ -157,13 +167,20 @@ func main() {
 				}
 				return
 			}
-			if input == "exit" {
+			if input == ".exit" {
+				return
+			}
+			if strings.Index(input, ".dump") == 0 {
+				path := strings.TrimSpace(input[5:])
```

Comment:
Should we validate the path first, so we don't end up passing something weird to the rest?
Otherwise:
```
.dump
```
At least tries to dump to a file name "" which certainly not quite what we want.
Also the fact that:
```
.dumppath
```
dumps to a file called "path" seems pretty weird.
We probably want to do a bit more with parsing of the line.